packages/core/test/collectAsyncCalls.ts

```diff
@@ -28,3 +28,34 @@ export function collectAsyncCalls<F extends jasmine.Func>(
     fail(`Unexpected extra call for spec '${currentSpec!.fullName}'`)
   }
 }
+export function asyncCollectAsyncCalls<F extends jasmine.Func>(
```

Comment:
**💭 thought:** ‏Not a big fan of that function name. Maybe we could use the same function as above, but also return a Promise

```ts
export function collectAsyncCalls<F extends jasmine.Func>(
  spy: jasmine.Spy<F>,
  expectedCallsCount: number,
  callback?: (calls: jasmine.Calls<F>) => void
): Promise<jasmine.Calls<F>> {
  ...
}
```

and deprecate `callback` usage so we can remove it one day.

Reply:
Done!